mappings/net/minecraft/util/math/MathHelper.mapping

```diff
@@ -503,3 +503,18 @@ CLASS net/minecraft/class_3532 net/minecraft/util/math/MathHelper
 		ARG 0 value
 		ARG 2 min
 		ARG 4 max
+	METHOD method_53063 lerpPositive (FII)I
```

Comment:
We should rename the 2 into `lerpFloor` and `lerpCeil` to indicate their fraction behaviors more accurately.

Reply:
This does not ceil values (e.g. 2.1 is still 2 in both methods), they just treat zero differently.

Reply:
After a review, this one is like "linear interpolate, but `delta` have to match exactly 0.0 and 1.0 for the 2 ends to be returned", so it slices the interval into `end - start - 1` parts. The regular lerp slices the interval into `end - start` parts, so it can return `start` when `delta != 0.0` but can only return `end` when `delta == 1.0`.